src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/Imaging/PropVariant.cs

```diff
@@ -364,7 +364,7 @@ internal void Init(object value)
                 else if (value is char)
                 {
                     varType = (ushort)VarEnum.VT_LPSTR;
-                    pszVal = Marshal.StringToCoTaskMemAnsi(new String(new char[] { (char)value }));
+                    pszVal = Marshal.StringToCoTaskMemAnsi(new String(stackalloc char[] { (char)value }));
```

Comment:
Does ((char)value) already have this kind of optimization ? If yes, I think we should use it, if not I think its fine to use stackalloc.

Reply:
I switched it to use `char.ToString((char)value)`.